pkgs/top-level/all-packages.nix

```diff
@@ -14547,6 +14538,7 @@ with pkgs;
   # The current default kernel / kernel modules.
   linuxPackages = linuxPackages_4_14;
   linux = linuxPackages.kernel;
+  cryptodev = linuxPackages.cryptodev;
```

Comment:
This marked as broken for our current kernel, which newer then 4.13.
Should we upgrade this?